AppBuilder/ABFactory.js

```diff
@@ -967,6 +967,29 @@ class ABFactory extends ABFactoryCore {
    isString(...params) {
       return _.isString(params);
    }
+
+   getCurrentDateValue() {
```

Comment:
I'm thinking it would be better to move this up to the other date functions ( lines 429 - 512).

And add a method header comment block describing what it is meant to do.  And why we would use it.

Reply:
Thank you for your suggestion.